src/Tinfoil/Digest.hs

```diff
@@ -0,0 +1,14 @@
+{-# LANGUAGE NoImplicitPrelude #-}
+{-# LANGUAGE OverloadedStrings #-}
+module Tinfoil.Digest(
```

Comment:
Yep, I wasn't sure about `Digest` but couldn't think of anything better at the time. Will go with `Encode`.